src/LinkDotNet.NCronJob/Configuration/NCronJobOptionBuilder.cs

```diff
@@ -61,21 +63,90 @@ public NCronJobOptionBuilder<T> AddJob<T>(Action<JobOptionBuilder>? options = nu
         return new NCronJobOptionBuilder<T>(Services, Settings);
     }
 
-    private static CronExpression GetCronExpression(JobOption option)
+    /// <summary>
+    /// Adds a job using an asynchronous anonymous delegate to the service collection that gets executed based on the given cron expression.
+    /// </summary>
+    /// <param name="jobDelegate">The delegate that represents the job to be executed.</param>
+    /// <param name="cronExpression">The cron expression that defines when the job should be executed.</param>
+    public NCronJobOptionBuilder AddJob(Delegate jobDelegate, string cronExpression)
```

Comment:
Just from an API point of view: If we also call that `AddNCronJob` would this be fitting?

My reasoning is that the user has one simple entry point.

Reply:
I think you mean the method in `ServiceCollectionExtensions`. I've updated the name to be more consistent with the entry point.
```csharp
    public static IServiceCollection AddNCronJob(this IServiceCollection services, Delegate jobDelegate, string cronExpression)
    {
        services.AddNCronJob(builder =>
            builder.AddJob(jobDelegate, cronExpression)
        );

        return services;
    }
    ```